pkg/utils/utils_virtual.go

```diff
@@ -93,55 +93,44 @@ type OSPNetworkData struct {
 	// Omit Services
 }
 
-func metaData(platformType PlatformType, address string) (netFilter string, macAddress string) {
-	switch platformType {
-	case VirtualOpenStack:
-		metaData, networkData := readOpenstackMetaData()
-		netFilter, macAddress = parseOpenstackMetaData(address, metaData, networkData)
-
-	default:
-		glog.V(2).Infof("Unknown PlatformType: %v", platformType)
-	}
-
-	return
-}
-
-func readOpenstackMetaData() (metaData *OSPMetaData, networkData *OSPNetworkData) {
-	networkData = &OSPNetworkData{}
-
-	rawBytes, err := ioutil.ReadFile(ospNetworkData)
+// ReadOpenstackMetaData reads the meta data from the openstack metadata file
+func ReadOpenstackMetaData() (metaData OSPMetaData, err error) {
```

Comment:
can we add a glog.info with the name of the functions like we have in other functions